gxaircom.py

```diff
@@ -0,0 +1,47 @@
+import arrow
+import requests
+
+from winds_mobi_provider import Provider, StationStatus, ureg, Q_, Pressure
+
+
+class GxAircom(Provider):
+    provider_code = "gxaircom"
+    provider_name = "gxaircom.com"
+
+    def process_data(self):
+        self.log.info("Processing GxAircom data...")
+        data = requests.get("http://www.mgenet.at/gxaircom/stations.php", timeout=(self.connect_timeout, self.read_timeout)).json()
+        for station in data:
+            winds_station = self.save_station(
+                provider_id=station["stationId"],
+                short_name=station["stationName"],
+                name=station["stationName"],  # Lets winds.mobi provide the full name with the help of Google Geocoding API
```

Comment:
If you want to let winds.mobi to guess the full name based on geolocation, pass name=None, like in the example of the README.md.

Reply:
When I do this I got the following error:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/opt/project/providers/gxaircom.py", line 47, in <module>
    gxaircom()
  File "/opt/project/providers/gxaircom.py", line 43, in gxaircom
    GxAircom().process_data()
  File "/opt/project/providers/gxaircom.py", line 15, in process_data
    winds_station = self.save_station(
  File "/opt/project/winds_mobi_provider/provider.py", line 384, in save_station
    raise ProviderException(
winds_mobi_provider.provider.ProviderException: Unable to determine station 'name': ProviderException('Google Geocoding API ZERO_RESULTS')